packages/connectivity_plus/connectivity_plus/pubspec.yaml

```diff
@@ -7,7 +7,9 @@ issue_tracker: https://github.com/fluttercommunity/plus_plugins/labels/connectiv
 
 environment:
   sdk: ">=2.18.0 <4.0.0"
-  flutter: ">=3.3.0"
+  # Flutter versions prior to 3.7 did not support the
+  # sharedDarwinSource option.
+  flutter: ">=3.7.0"
```

Comment:
@vbuberen What do you think about increasing the min flutter version in order to share the macos and ios codebases? personally I am ok with it, but want to know your thoughts

Reply:
I am Ok with that here, even though I don't like this merging of iOS and MacOS into one.

Reply:
I opted to maintain a single codebase for both platforms, aligning with best practices for Apple's products.

Reply:
I am in favor of having a single code folder for both platform when possible, although it may not be the case for all plugins.